app/static/scripts/navmenu.js

```diff
@@ -0,0 +1,14 @@
+function toggleNav() {
+    var nav = document.getElementById("toggle-menu");
+    var icon = document.getElementById("icon");
+    if (nav.style.right === "0px") {
```

Comment:
This could be written in a bit more cleaner way if you move the condition to a variable, e.g.:

```
const isOpen = nav.style.right === "0px";
nav.style.right = isOpen ? "-50%" : "0px";
```

And then to remove/add classList you could instead use `classList.toggle(<class-name>, isOpen)`.

This should remove the if-else and make the code look a bit more cleaner imo

Reply:
You are completely right, thank you very much for the suggestion, I will improve it immediately! 🚀